src/matchers/utils.ts

```diff
@@ -124,15 +124,23 @@ export const getElementText = async (
 export const quote = (val: string | null) => (val === null ? "" : `'${val}'`)
 
 export const getMessage = (
-  { isNot, promise, utils }: jest.MatcherContext,
+  { isNot, promise, utils, expand }: jest.MatcherContext,
   matcher: string,
   expected: boolean | string | number | null,
   received: boolean | string | number | null
 ) => {
   const message = isNot
```

Comment:
idk, but these nested ternary are difficult to read, maybe we can rewrite it?
Something like this:
```js
const getMessage = () => {
  isNot
    ? `Expected: not ${utils.printExpected(expected)}`
    : `Expected: ${utils.printExpected(expected)}\n` +
      `Received: ${utils.printReceived(received)}`;
};

typeof expected === "boolean"
  ? getMessage()
  : utils.printDiffOrStringify(
      expected,
      received,
      "Expected",
      "Received",
      expand
    );

```

Reply:
Yeah, I can rewrite it to use if statements

Reply:
@mskelton wouldn't it work with my suggestion? Sorry, i don't have a chance to check it right now

Reply:
So, actually it turns out to be even easier since `printDiffOrStringify` will work just fine for boolean as well.  I was just over complicating it!